### DISCUSSION TITLE
v0.11.0-rc1

### DISCUSSION THREAD

**Discussion by weltenwort** [weltenwort/frigate-synology-dsm7#60] [Announcements]
:warning: **ATTENTION:** this minor version release changes the database structure in a way that prevents downgrades to prior versions

## Container images
* [ghcr.io/weltenwort/frigate-synology-dsm7:v0.11.0-rc1](https://github.com/weltenwort/frigate-synology-dsm7/pkgs/container/frigate-synology-dsm7/30000401?tag=v0.11.0-rc1)

## Significant changes
* update to frigate-0.11.0-rc1 by @baylanger in https://github.com/weltenwort/frigate-synology-dsm7/pull/59
* Update docker-compose.yml by @Caros2017 in https://github.com/weltenwort/frigate-synology-dsm7/pull/55
* Fix typo in docker-compose invocation by @weltenwort in https://github.com/weltenwort/frigate-synology-dsm7/pull/51

## CI changes
* Bump docker/build-push-action from 3.0.0 to 3.1.0 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/57
* Bump sigstore/cosign-installer from 2.4.1 to 2.5.0 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/58

## New Contributors
* @Caros2017 made their first contribution in https://github.com/weltenwort/frigate-synology-dsm7/pull/55
* @baylanger made their first contribution in https://github.com/weltenwort/frigate-synology-dsm7/pull/59

**Full Changelog**: https://github.com/weltenwort/frigate-synology-dsm7/compare/v0.11.0-beta5...v0.11.0-rc1

<hr /><em>This discussion was created from the release <a href='https://github.com/weltenwort/frigate-synology-dsm7/releases/tag/v0.11.0-rc1'>v0.11.0-rc1</a>.</em>

**Comment by imne** [comment]
great work on this, please keep up the good work
